src/test/java/uk/gov/companieshouse/pscdataapi/api/ChsKafkaApiServiceTest.java

```diff
@@ -58,8 +64,9 @@ void invokeChsKafkaEndpoint() throws ApiErrorResponseException {
         Assertions.assertThat(apiResponse).isNotNull();
 
         verify(internalApiClient, times(1)).privateChangedResourceHandler();
-        verify(privateChangedResourceHandler, times(1)).postChangedResource(Mockito.any(), Mockito.any());
+        verify(privateChangedResourceHandler, times(1)).postChangedResource(Mockito.any(), changedResourceCaptor.capture());
         verify(privateChangedResourcePost, times(1)).execute();
+        Assertions.assertThat(changedResourceCaptor.getValue()).isNotNull();
```

Comment:
Could be more specific here rather than checking for NotNull.
E.g 
`Assertions.assertThat(changedResourceCaptor.getValue().getResourceUri()).isEqualTo(<expected uri>)`